test/kitchen/tasks/run-test-kitchen.sh

```diff
@@ -90,12 +100,14 @@ export MAJOR_VERSION=$2
 # on linux it can just download the latest version from the package manager
 if [ -z ${AGENT_VERSION+x} ]; then
   pushd ../..
-    export AGENT_VERSION=`inv agent.version --url-safe --git-sha-length=7 --major-version $MAJOR_VERSION`
-    export DD_AGENT_EXPECTED_VERSION=`inv agent.version --url-safe --git-sha-length=7 --major-version $MAJOR_VERSION`
+    AGENT_VERSION=$(inv agent.version --url-safe --git-sha-length=7 --major-version "$MAJOR_VERSION")
+    export AGENT_VERSION
+    DD_AGENT_EXPECTED_VERSION=$(inv agent.version --url-safe --git-sha-length=7 --major-version "$MAJOR_VERSION")
+    export DD_AGENT_EXPECTED_VERSION
   popd
 fi
 
-invoke -e kitchen.genconfig --platform=$KITCHEN_PLATFORM --osversions=$KITCHEN_OSVERS --provider=azure --testfiles=$1
+invoke -e kitchen.genconfig --platform="$KITCHEN_PLATFORM" --osversions="$KITCHEN_OSVERS" --provider="$KITCHEN_PROVIDER" --arch="$KITCHEN_ARCH" --testfiles="$1"
```

Comment:
Regular kitchen tests (the ones in the `kitchen_testing` stage) fail with:
```
tasks/run-test-kitchen.sh: line 110: KITCHEN_ARCH: unbound variable
```

We need to either define `KITCHEN_ARCH` in all kitchen jobs, or set a default value of `x86_64` for this parameter in this script.

Reply:
Added a default value.